lib/active_record/connection_adapters/percona_adapter.rb

```diff
@@ -138,6 +138,14 @@ def schema_creation
       def error_number(_exception); end
 
       def full_version
+        if ActiveRecord::VERSION::MAJOR < 6
+          mysql_adapter.raw_connection.server_info[:version]
```

Comment:
[nit] Should this call `get_full_version` to avoid duplication?

Reply:
Nice catch! I've fixed this in 64c2116.